src/System.Text.Json/tests/Utf8JsonReaderTests.MultiSegment.cs

```diff
@@ -1124,29 +1128,6 @@ public static void JsonWithUnfinishedMultiLineCommentNonFinalBlockMultiSegment(s
             Assert.Equal(0, json.BytesConsumed);
         }
 
-        [Theory]
-        [InlineData("//", "", 1)]
-        [InlineData("//", "", 2)]
-        [InlineData("//", "", 3)]
-        [InlineData("//", "", 100)]
-        [InlineData("//a", "a", 1)]
-        [InlineData("//a", "a", 2)]
-        [InlineData("//a", "a", 3)]
-        [InlineData("//a", "a", 100)]
-        [InlineData("//abc", "abc", 1)]
-        [InlineData("//abc", "abc", 2)]
-        [InlineData("//abc", "abc", 3)]
-        [InlineData("//abc", "abc", 100)]
-        public static void JsonWithSingleLineCommentWithNoLineEndingsNonFinalBlockMultiSegment(string jsonString, string expectedComment, int segmentSize)
```

Comment:
looks like you didn't like this test?

Reply:
The second argument isn't used.  When you remove that, it's an exact duplicate of another existing test.